### PR TITLE
Fix powerups

### PR DESCRIPTION
**Pull request by keith-gan**

### CODE REVIEW

File: TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift [hunk 1]
```diff
@@ -5,7 +5,16 @@
 //  Created by Keith Gan on 31/3/24.
 //
 
+import Foundation
+
 class InvulnerabilityPowerUp: EventTransformation {
+    let DURATION = CGFloat(5)
+    let id: UUID
+
+    init() {
```
Comment (Vanessamae23) [inline TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift:14]: is the `id` here the same as the `id` in `EventTransformation` ? Is it necessary to repeat. Also should we instead make it extend identifiable so we enforce the need to have id if we are using it

**Review by Vanessamae23 — COMMENTED** [review]
Just a bit of a comment to clarify

File: TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift (hunk 1 above)
Comment (keith-gan) [reply TowerForge/TowerForge/GameModule/PowerUps/InvulnerabilityPowerUp.swift:14]: - this is an `EventTransformation` so the `id` is the same
- added the conformance to `Identifiable` for `EventTransformation`

**Review by Vanessamae23 — APPROVED** [review]
lgtm

**Pull request merged**